frontend/src/main/services/courseSearches.js

```diff
@@ -0,0 +1,34 @@
+import fetch from "isomorphic-unfetch";
+
+ const fetchBasicCourseJSON = async (_event, fields) => {
```

Comment:
Fix formatting (remove indentations)

Reply:
Instead of `fetch` we are using `useBackend` these days, which uses `axios` under the hood, like we did in the team03 assignments. 

I know that the S21 code may have used `fetch`, but we are trying to standardize on a smaller set of dependencies.